Saule/Http/JsonApiConfiguration.cs

```diff
@@ -22,6 +22,11 @@ public class JsonApiConfiguration
         /// <summary>
         /// Gets the expressions that are used to evaluate filter queries on a per-type basis.
         /// </summary>
-        public QueryFilterExpressionCollection QueryFilterExpressions { get; } = new QueryFilterExpressionCollection();
+        public QueryFilterExpressionCollection QueryFilterExpressions { get; } = new QueryFilterExpressionCollection();
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the new experimental graph serializer is used.
+        /// </summary>
+        public bool UseGraphSerializer { get; set; } = false;
```

Comment:
I wonder if we actually need this; I would prefer we just swap out the serializer (provided that all the tests pass with the new one).

I don't think anyone would want to use the old serializer with this new one in place anyway.